### ISSUE TITLE
Navigation: Update `aria-current` to `page`

### ISSUE THREAD

**Issue by marvieqa** [CodeForPhilly/clean-and-green-philly#660]
## Description

Update the `<aria-current="true">` value to `page` for Navigation linked "Find Properties", "Get Access", "Transform" & "About", because it is more appropriate.

## Acceptance Criteria

Update `aria-current` value to `page` to the following items:
- [ ] Find Properties 
- [ ] Get Access
- [ ] Transform
- [ ] About

**Comment by KenMain5** [comment]
I can work on this. Can someone assign this to me?

**Comment by CodeWritingCow** [comment]
@KenMain5 it's yours!

**Comment by CodeWritingCow** [comment]
@KenMain5 any progress on this ticket?

**Comment by KenMain5** [comment]
Working on it. All should be done by tomorrow afternoon mb
